ui/src/main/resources/static/components/browse-filter/browse-filter-faq.html

```diff
@@ -0,0 +1,18 @@
+<html>
+   <head>Browser filter FAQ:</head>
+   <body>
+    
+       <p>Filter how to:</p>
+       <ul>
+           <li><p>List of columns has been populated after a first fetch. It contains by all fetched columns.</p></li>
```

Comment:
I'm not pro, but I guess it would be more correct :)
List of columns is populated after the first fetch. It contains all the fetched columns
...
In case the columns you have chosen... This notation is applicable for "eq....
...
Filter operates on only a set of records, which KM managed to read for given timeout....
If the set of record does not contain desired data, just ...